### PR TITLE
add ChrRaces.md

### PR DESCRIPTION
**Pull request by pangolp**
## Description

Adding the missing file

## Issues Addressed:
https://github.com/azerothcore/wiki/issues/198

### CODE REVIEW

**Comment by Kitzunu** [comment]
Rename the file to chrraces

**Comment by pangolp** [comment]
> Rename the file to chrraces

For any particular reason? The original file is in Title. But maybe there is some reason I don't know and I would like to know it.

**Comment by Kitzunu** [comment]
yea, uppercase in URLs are bad and hard to work with for the end-user.

that is why all other files are only lowercase as well.

**Comment by pangolp** [comment]
> yea, uppercase in URLs are bad and hard to work with for the end-user.
> 
> that is why all other files are only lowercase as well.

Perfect, now I modify it. I thought I saw them in capital letters, but it's true about the URL.

**Comment by pangolp** [comment]
Damn Windows, I didn't take the name change, I had to delete it and add it again, but this time in lowercase xD.

**Comment by Kitzunu** [comment]
It should be the same, but with this wiki being built on JS (if I remember correctly) this is not possible to do, already discussed it with Yehonal on the main wiki repo

**Comment by pangolp** [comment]
> It should be the same, but with this wiki being built on JS (if I remember correctly) this is not possible to do, already discussed it with Yehonal on the main wiki repo

I didn't understand this. Anyway, I already renamed the file.

**Comment by Kitzunu** [comment]
In the "real world" /ChrRaces == /chrraces (for example google.com/GMAIL == google.com/gmail)

Case should not matter, but this is not possible with what the wiki is build on

**Comment by pangolp** [comment]
> n the "real world" /ChrRaces == /chrraces (for example google.com/GMAIL == google.com/gmail)
> 
> Case should not matter, but this is not possible with what the wiki is build on

Yes, I understand. Also sometimes slug, based on the title, is used. I have used it in some frameworks like Django for example.

File: docs/chrraces.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+| 4  | 3     | 8             | Night Elf    |
+| 5  | 4     | 16            | Undead       |
+| 6  | 5     | 32            | Tauren       |
+| 7  | 6     | 64            | Gnome        |
+| 8  | 7     | 128           | Troll        |
+| 9  | 8     | 256           | Goblin (4.x) |
+| 10 | 9     | 512           | Blood Elf    |
+| 11 | 10    | 1024          | Draenei      |
+
+![raceMask](https://user-images.githubusercontent.com/2810187/104961176-909b1500-59b4-11eb-8a55-2db891410ba7.png)
```
Comment (Kitzunu) [inline docs/chrraces.md:79]: I don't know if this is necessary, it is basic bit work and it is explained in another page https://www.azerothcore.org/wiki/Bit-and_bytes-tutorial

Comment (pangolp) [reply docs/chrraces.md:79]: > I don't know if this is necessary, it is basic bit work and it is explained in another page https://www.azerothcore.org/wiki/Bit-and_bytes-tutorial

I did not know about the existence of this file.
We could then create a link within the document.

Comment (pangolp) [reply docs/chrraces.md:79]: It has the b in capital letters xD.

Comment (Kitzunu) [reply docs/chrraces.md:79]: I know, sucks.

But I still think it is not necessary. Perhaps add something along the lines to "To use multiple races you add the bit values together` but nothing more. I don't think it is necessary to explain it :)

**Comment by pangolp** [comment]
That's it, it's been edited. @Kitzunu

File: docs/chrraces.md (hunk 1 above)
Comment (pangolp) [reply docs/chrraces.md]: I already added the link to the document you mentioned

File: docs/chrraces.md [hunk 2]
```diff
@@ -60,20 +60,4 @@ This dbc contains all possible races, some of which are unused and unavailable t
...
-| 5  | 4     | 16            | Undead       |
-| 6  | 5     | 32            | Tauren       |
-| 7  | 6     | 64            | Gnome        |
-| 8  | 7     | 128           | Troll        |
-| 9  | 8     | 256           | Goblin (4.x) |
-| 10 | 9     | 512           | Blood Elf    |
-| 11 | 10    | 1024          | Draenei      |
-
-![raceMask](https://user-images.githubusercontent.com/2810187/104961176-909b1500-59b4-11eb-8a55-2db891410ba7.png)
+For more information, on how to obtain the values, you can consult the following link: [bit-and_bytes-tutorial](Bit-and_bytes-tutorial)
```
Comment (Kitzunu) [inline docs/chrraces.md:63]:
```suggestion
If you want to learn how bits work you can read the tutorial here [bit-and_bytes-tutorial](Bit-and_bytes-tutorial)
```

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**